docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -114,14 +114,66 @@ Complete list of `SQL Server data types
 
 .. include:: jdbc-type-mapping.fragment
 
+.. _sqlserver-sql-support:
+
 SQL support
 -----------
 
-The following SQL statements are not yet supported:
-
-* :doc:`/sql/delete`
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
+The connector provides **read access and limited write access** to data and
+metadata in SQL Server. The following section lists all supported SQL statements
+and provides further details.
+
+.. important::
+  No other SQL statements are supported.
+
+Read access:
+
+* :doc:`/sql/select` including:
+
+ * :doc:`/sql/match-recognize`
+ * :doc:`/sql/values`
+
+* :doc:`/sql/show-catalogs`
+* :doc:`/sql/show-columns`
+* :doc:`/sql/show-schemas`
+* :doc:`/sql/show-tables`
+* :doc:`/sql/describe`
+
+Write access:
+
+* :doc:`/sql/create-schema`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/drop-table`
```

Comment:
Requires setting `allow-drop-table` in the JDBC connector config.

Reply:
I believe Jessie212 is planning to add that info

Reply:
I'm confused by this resolution. @Jessie212 will add the fact that the drop-table command requires the allow-drop-table=true config setting ... where? In this file? When? I hope not in another file. This appears to be the right place to say that this command is supported but requires a setting to activate it.

Reply:
My 2 cents, what I've seen in other supported SQL statements lists is a ref link here to a section down below in the same page that explains in more detail. So I'd expect it to be the same file, but maybe a standardized blurb between each JDBC connector for consistency.

Reply:
@Ordinant, @jhlodin is correct. I'm going to include it as a fragment.